test/unit/sql/add-column.test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+/* jshint -W030, -W110 */
+var Support   = require(__dirname + '/../support')
+  , DataTypes = require('../../../lib/data-types')
+  , expectsql = Support.expectsql
+  , sinon = require('sinon')
+  , current   = Support.sequelize
+  , Promise = current.Promise;
+
+
+if (current.dialect.name === 'mysql') {
+  describe(Support.getTestDialectTeaser('SQL'), function() {
+    describe('addColumn', function () {
+
+      var Model = current.define('users', {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true
+        },
+        level_id: {
+          type: DataTypes.INTEGER
+        }
+      }, { timestamps: false });
+
+      before(function () {
+
+        this.stub = sinon.stub(current, 'query', function (sql) {
+          return Promise.resolve(sql);
+        });
+      });
+
+      beforeEach(function () {
+        this.stub.reset();
+      });
+
+      after(function () {
+        this.stub.restore();
+      });
+
+      it('properly generate alter queries', function(){
+        return current.getQueryInterface().addColumn(Model.getTableName(), 'level_id', {
+          type: DataTypes.FLOAT,
+          allowNull: false,
+        }).then(function(sql){
+          expectsql(sql, {
+            mysql: 'ALTER TABLE `users` ADD `level_id` FLOAT NOT NULL;',
+          });
+        });
+      });
```

Comment:
can you please refactor this unit test to not rely on stubbing but simply test the query interface directly?
(goes for both tests really)